<!-- launch file for running cam_esposure_controller with ROS images -->


<launch>

  <!-- node for saving video file -->
  <node pkg="misc_pkg" type="cam_exposure_controller.py" name="exposure_controller">
    <param name="intensity_threshold" value="50.0" />
    <param name="intensity_desired" value="120.0" />
    <param name="k_gain" value="0.5" />
    <param name="minimum_exposure" value="-3.8" />
    <param name="maximum_exposure" value="0.0" />
    <param name="camera_node" value="pointgrey_camera_node" />
  </node>


</launch>
